utils.js

```diff
@@ -184,18 +181,18 @@ async function makeReactLayout() {
 exports.makeReactLayout = makeReactLayout;
 
 if (require.main === module) {
-    makeReactLayout().then(data => {
-        const fs = require('fs');
-        const {jsxLines, cssLines, manifest} = data;
+    (async function () {
```

Comment:
this looks like a verbatim copy of `onPreBootstrap`, maybe call that function instead

Reply:
the idea was so that any time for dev you could just `node utils.js` and get the layout file generated, I didn't want to pull in all of gatsby.

I think I'll switch it around though, rename the file to mklayout.js, add a saveLayout() or something, and make onPreBootstrap call saveLayout